blog/2024-07-30-how-to-run-webassembly-components-on-kubernetes-with-wasmcloud/index.mdx

```diff
@@ -0,0 +1,182 @@
+```
+Next install the upstream NATS Helm chart:
+
+```shell
+helm upgrade --install -f https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/nats-values.yaml nats nats/nats
+```
+You can validate the NATS installation with:
+
+```shell
+kubectl rollout status deploy,sts -l app.kubernetes.io/instance=nats
+```
+Deploy wadm to your Kubernetes cluster with a Helm chart hosted by wasmCloud: 
+
+```shell
+helm install wadm -f https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/wadm-values.yaml oci://ghcr.io/wasmcloud/charts/wadm
+```
+Validate the wadm installation with:
+
+```shell
+kubectl rollout status deploy -l app.kubernetes.io/instance=wadm
+```
+Once NATS and wadm are ready, you can move on to deploy the operator.
+
+#### 2. Deploy the operator
+
+To deploy the operator:
+
+```shell
+kubectl apply -k https://github.com/wasmCloud/wasmcloud-operator/deploy/base
+```
+Validate the installation by checking that the pods are ready...
+
+```shell
+kubectl rollout status deploy -l app=wasmcloud-operator -n wasmcloud-operator
+```
+...and confirming that the API service is available:
+```shell
+kubectl get apiservices.apiregistration.k8s.io v1beta1.core.oam.dev
+```
+
+#### 3. Create wasmCloud hosts
+
+Our final step is to apply the `wasmcloud-host` manifest:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/wasmcloud-host.yaml
+```
+Check the status of your wasmCloud host:
+
+```shell
+kubectl describe wasmcloudhostconfig wasmcloud-host
+```
+
+At this point, you're up and running. Let's try running some WebAssembly components with our new wasmCloud installation. 
+
+#### Run a WebAssembly component on Kubernetes
+
+When you `kubectl apply` a [wasmCloud application manifest](../../docs/ecosystem/wadm/model), the cluster will automatically provision the component workload with wasmCloud. We'll use the [`hello-world-application.yaml` manifest](https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/hello-world-application.yaml) included in the operator's quickstart. (You can check out the [source code for the application itself](https://github.com/wasmCloud/wasmCloud/tree/main/examples/rust/components/http-hello-world) in the wasmCloud monorepo.)
+
+Before we launch the application, let's take a quick look at an excerpt of the manifest:
+
+```yaml {7-7}
+...
+spec:
+  components:
+    - name: http-component
+      type: component
+      properties:
+        image: ghcr.io/wasmcloud/components/http-hello-world-rust:0.1.0
+...
+```
+On line 7, you can see that we're deploying our component from an image. Specifically, the component is packaged as an OCI artifact. These aren't containers, but pure components conforming to OCI standards&mdash;meaning you can use the same registry for component artifacts that you're already using for container images. 
```

Comment:
You're most welcome to leave this nuance out if it doesn't make sense here, I just figured that it could be nice to link readers to the definition in case they are interested to learn more or hadn't heard of this development yet